src/main/app/controller/courts/AddressController.ts

```diff
@@ -13,15 +13,18 @@ import {
 import {CourtAddressPageData} from '../../../types/CourtAddressPageData';
 import {SelectItem} from '../../../types/CourtPageData';
 import {postcodeIsValidFormat} from '../../../utils/validation';
+import {County} from '../../../types/County';
 
 @autobind
 export class AddressController {
 
   getAddressTypesError = 'A problem occurred when retrieving the court address types.';
   getAddressesError = 'A problem occurred when retrieving the court addresses.';
+  getCountyError = 'A problem occurred when retrieving the counties';
   updateAddressError = 'A problem occurred when saving the court addresses.';
   multipleVisitAddressError = 'Only one visit address is permitted.';
   typeRequiredError = 'Address Type is required.';
+  countyRequiredError = 'County is required. ';
```

Comment:
Are counties required? If someone lives in the centre of Birmingham, what would the county be?

Reply:
yes counties are required and for someone living in Birmingham would select West Midlands as a county